canonical-web-and-design/snapcraft.io
# No loop after youtube video finishes

### QA steps
1. Pull the branch or runt the demo service
2. Visit a snap with a youtube video like /code
3. Wait for the video to finish and video should not loop at the end
4. #winning
